appinventor/appengine/src/com/google/appinventor/client/actions/DownloadAction.java

```diff
@@ -0,0 +1,54 @@
+package com.google.appinventor.client.actions;
```

Comment:
Ditto copyright

Reply:
Actually, it looks like this class isn't referenced from anywhere, so we should just delete it.